reactor-netty-core/src/main/java/reactor/netty/tcp/SslProvider.java

```diff
@@ -629,6 +631,30 @@ public void userEventTriggered(ChannelHandlerContext ctx, Object evt) {
 	}
 
 	static final Logger log = Loggers.getLogger(SslProvider.class);
+
+	/**
+	 * <a href="https://wiki.mozilla.org/Security/Server_Side_TLS#Modern_compatibility">Mozilla Modern Cipher
+	 * Suites</a> minus the following cipher suites that are black listed by the
+	 * <a href="https://tools.ietf.org/html/rfc7540#appendix-A">HTTP/2 RFC</a>.
+	 * Copied from io.netty.handler.codec.http2.Http2SecurityUtil
+	 */
+	static final List<String> HTTP2_TLS12_CIPHERS =
```

Comment:
it seems there's another unrelated change that creeped in the PR

Reply:
ah got it, it was using something from netty's http2 artifact previously and we don't want that dependency anymore 👍

Reply:
`reactor-netty-core` does not have dependency to `netty-codec-http2`
this change removes the usage of `Http2SecurityUtil` which is in `netty-codec-http2`